recipes/foxglove-websocket/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class FoxgloveWebSocketConan(ConanFile):
+    name = "foxglove-websocket"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/foxglove/ws-protocol"
+    description = "A C++ server implementation of the Foxglove WebSocket Protocol"
+    license = "MIT"
+    topics = ("foxglove", "websocket")
+
+    settings = ("os", "compiler", "build_type", "arch")
+    requires = ("nlohmann_json/[^3.10.4]", "websocketpp/[^0.8.2]")
+    generators = "cmake"
+
+    _source_root = "source_root"
+    _source_package_path = os.path.join(_source_root, "cpp", "foxglove-websocket")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_root)
+
+    def validate(self):
+        tools.check_min_cppstd(self, "17")
```

Comment:
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-is-a-toolscheck_min_cppstd-call-not-enough

Please see above and add the missing `if` 🙏

Reply:
Thanks, updated